client/state/sites/selectors/build-seo-title.js

```diff
@@ -1,17 +1,9 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 export default ( titleFormats, type, { site, post = {}, tag = '', date = '' } ) => {
 	const processPiece = ( piece = {}, data ) =>
-		'string' === piece.type ? piece.value : get( data, piece.type, '' );
+		'string' === piece.type ? piece.value : data[ piece.type ] ?? '';
 
 	const buildTitle = ( format, data ) =>
-		get( titleFormats, format, [] ).reduce(
-			( title, piece ) => title + processPiece( piece, data ),
-			''
-		);
+		( titleFormats[ format ] ?? [] ).map( ( piece ) => processPiece( piece, data ) ).join( '' );
```

Comment:
Any chance for `titleFormats` to be `undefined`?

Reply:
By tracing the calls (there are two or three in total) I see that the worst case is that `titleFormats` is an empty `{}` object. Never `null` or `undefined`.